woocommerce-gateway-ebanx/gateways/class-wc-ebanx-debit-card-gateway.php

```diff
@@ -132,7 +138,47 @@ protected function save_order_meta_fields($order, $request)
 		parent::save_order_meta_fields($order, $request);
 
 		update_post_meta($order->id, '_cards_brand_name', $request->payment->payment_type_code);
-		update_post_meta($order->id, '_masked_card_number', $_POST['ebanx_masked_card_number']);
+		update_post_meta($order->id, '_masked_card_number', $_POST['ebanx_debit_masked_card_number']);
+	}
+
+	/**
+	 * Save user's meta fields for future use
+	 *
+	 * @param  WC_Order $order The order created
+	 * @return void
+	 */
+	protected function save_user_meta_fields($order)
+	{
+		parent::save_user_meta_fields($order);
+
+		if ($this->userId && $this->configs->settings['save_card_data'] === 'yes' && isset($_POST['ebanx-save-debit-card']) && $_POST['ebanx-save-debit-card'] === 'yes') {
```

Comment:
We're not using superglobals anymore, please replace `isset($_POST[<value>])` with `WC_EBANX_Request::has(<value>)`